### PR TITLE
Doc: Departures example added in navitia.io documentation

### PR DESCRIPTION
**Pull request by lrocheWB**
- Navitia.io documentation updated (curl requests added in departures section)

Result:
screen-departures-example-3

### CODE REVIEW

**Comment by Netmisa** [comment]
I think it's better to see what this request return for this example. (Like other one)

**Comment by stifoon** [comment]
Agree with @Netmisa : take a look at https://github.com/CanalTP/navitia/pull/1646 for example

**Comment by lrocheWB** [comment]
Done

File: documentation/slate/source/includes/apis.md [hunk 1]
```diff
@@ -1028,6 +1028,60 @@ nop      | data_freshness   | enum                           | Define the freshn
 <a name="departures"></a>Departures
 -----------------------------------
 
+``` shell
+
+#using "headers"
+$ curl 'https://api.navitia.io/v1/coverage/sandbox/lines/line:RAT:M1/departures?from_datetime=20160615T1337' -H 'Authorization: 3b036afe-0110-4202-b9ed-99718476c2e0'
+
+HTTP/1.1 200 OK
```
Comment (stifoon) [inline documentation/slate/source/includes/apis.md:1037]: You should add "#Request" and "#Response" comments, as it is done in the other example:
`#Request`
`$ curl 'https://api.navitia.io/v1/coverage/sandbox/lines/line:RAT:M1/departures?from_datetime=20160615T1337' -H 'Authorization: 3b036afe-0110-4202-b9ed-99718476c2e0'`

`#Response`
`HTTP/1.1 200 OK`

File: documentation/slate/source/includes/apis.md [hunk 2]
```diff
@@ -1028,6 +1028,60 @@ nop      | data_freshness   | enum                           | Define the freshn
 <a name="departures"></a>Departures
 -----------------------------------
 
+``` shell
+
+#using "headers"
```
Comment (stifoon) [inline documentation/slate/source/includes/apis.md]: I think it is a copy/paste comment from authentication :)

Comment (lrocheWB) [inline documentation/slate/source/includes/apis.md]: yep

File: documentation/slate/source/includes/apis.md (hunk 1 above)
Comment (lrocheWB) [inline documentation/slate/source/includes/apis.md:1037]: Done

**Pull request merged**